test/NinjaTools.FluentMockServer.API.Tests/Configuration/ValidConfigurationFileData.cs

```diff
@@ -0,0 +1,104 @@
+
+namespace NinjaTools.FluentMockServer.API.Tests.Configuration
+{
+    public class ValidConfigurationFileData : TheoryData<IConfigurationFile, string, string>
+    {
+        public ValidConfigurationFileData()
+        {
+            Add(new ConfigurationFile()
+                {
+                    new Setup
+                    {
+                        Matcher = new RequestMatcher()
+                        {
+                            Headers = new Dictionary<string, string[]>
+                            {
+                                {"Content-Type", new[] {"application/json", "application/cloudevents+json"}},
+                                {"Content-Length", new[] {"105"}}
+                            },
+                            Path = "/some/path",
+                            Method = "POST",
+                            BodyMatcher = new RequestBodyMatcher
+                            {
+                                Content = "request body to match \n over \n multiple \nlines",
+                                MatchExact = false,
+                                Type = RequestBodyType.Text
+                            },
+                            QueryString = new QueryString("?id=123")
+                        },
+                        Action = new ResponseAction
+                        {
+                            Response = new HttpResponse
+                            {
+                                Body = "some body \n over \n multiple \nlines",
+                                StatusCode = 200
+                            }
+                        }
+                    }
+                },
+                @"[
+  {
+    ""Id"": ""1"",
+    ""Matcher"": {
+      ""BodyMatcher"": {
+        ""Content"": ""request body to match \n over \n multiple \nlines"",
+        ""Type"": 0,
+        ""MatchExact"": false
+      },
+      ""Path"": ""/some/path"",
+      ""Method"": ""POST"",
+      ""Headers"": {
+        ""Content-Type"": [
+          ""application/json"",
+          ""application/cloudevents+json""
+        ],
+        ""Content-Length"": [
+          ""105""
+        ]
+      },
+      ""QueryString"": {
+        ""Value"": ""?id=123"",
+        ""HasValue"": true
+      }
+    },
+    ""Action"": {
+      ""Response"": {
+        ""StatusCode"": 200,
+        ""Body"": ""some body \n over \n multiple \nlines""
+      }
+    }
+  }
+]", @"- Id: 1
+  Matcher:
+    BodyMatcher:
+      Content: ""request body to match \n over \n multiple \nlines""
+      Type: Text
+      MatchExact: false
+    Path: /some/path
+    Method: POST
+    Headers:
+      Content-Type:
+      - application/json
+      - application/cloudevents+json
+      Content-Length:
+      - 105
+    QueryString:
+      Value: ?id=123
+      HasValue: true
+  Action:
+    Response:
+      StatusCode: 200
+      Body: ""some body \n over \n multiple \nlines""");
+        }
+
+        public const int ConfigurationFileIndex = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4765163)